roles/ansible/tower/launch-tower-jobs/tasks/launch-job.yml

```diff
@@ -0,0 +1,97 @@
+---
+- name: 'Get the job template id based on the job template name'
+  set_fact:
+    job_template_id: '{{ item.id | int }}'
+  when:
+    - item.name|trim == launch_job.job_template|trim
+  with_items:
+    - '{{ existing_job_templates.rest_output }}'
+        # Utilize the `rest_get` library routine to ensure REST pagination is handled
```

Comment:
is this comment perhaps a bit out of place? Was it meant for the next task/block?